sympy/calculus/util.py

```diff
@@ -112,3 +119,672 @@ def elm_domain(expr, intrvl):
+                    if other.min.is_negative:
+                        if other.max.is_zero:
+                            return Limits(-oo, self.min/other.min)
+                        if other.max.is_positive:
+                            # the actual answer is a Union of Limits,
+                            # Union(Limits(-oo, self.min/other.min),
+                            #       Limits(self.min/other.max, oo))
+                            return Limits(-oo, oo)
+
+                    if other.min.is_zero and other.max.is_positive:
+                        return Limits(self.min/other.max, oo)
+
+            elif other.is_real:
+                if other is S.Infinity or other is S.NegativeInfinity:
+                    if self == Limits(-oo, oo):
+                        return Limits(-oo, oo)
+                    if self.max is S.Infinity:
+                        return Limits(Min(0, other), Max(0, other))
+                    if self.min is S.NegativeInfinity:
+                        return Limits(Min(0, -other), Max(0, -other))
+                if other.is_positive:
+                    return Limits(self.min/other, self.max/other)
+                elif other.is_negative:
+                    return Limits(self.max/other, self.min/other)
+            return Mul(self, 1/other, evaluate=False)
+
+        return NotImplemented
+
+    __truediv__ = __div__
+
+    @_sympifyit('other', NotImplemented)
+    def __rdiv__(self, other):
+        if isinstance(other, Expr):
+            if other.is_real:
+                if other.is_zero:
+                    return S.Zero
+                if S.Zero in self:
+                    if self.min == S.Zero:
+                        if other.is_positive:
+                            return Limits(Mul(other, 1/self.max), oo)
+                        if other.is_negative:
+                            return Limits(-oo, Mul(other, 1/self.max))
+                    if self.max == S.Zero:
+                        if other.is_positive:
+                            return Limits(-oo, Mul(other, 1/self.min))
+                        if other.is_negative:
+                            return Limits(Mul(other, 1/self.min), oo)
+                    return Limits(-oo, oo)
+                else:
+                    return Limits(Min(other/self.min, other/self.max),
+                            Max(other/self.min, other/self.max))
+            return Mul(other, 1/self, evaluate=False)
+        else:
+            return NotImplemented
+
+    __rtruediv__ = __rdiv__
+
+    @_sympifyit('other', NotImplemented)
+    def __pow__(self, other):
+        from sympy.functions.elementary.miscellaneous import real_root
+        if isinstance(other, Expr):
+            if other is S.Infinity:
+                if self.min.is_nonnegative:
+                    if self.max < 1:
+                        return S.Zero
+                    if self.max == 1:
+                        return FiniteSet(0, 1)
```

Comment:
> Did i missed something?

We should not think of ``Limits(0, 1)`` as representing a *fixed* number ``a`` in the interval [0. 1].
Instead, we should keep in mind that it is used in connection with a function, as the *set* of its accumulation values (possibly coming arbitrarily close to 1).

This is quite analogous to ``1**oo``. In the docstring of ``Pow`` there is
>     | 1**oo        | nan     | Because there are various cases where         |
    | 1**-oo       |         | lim(x(t),t)=1, lim(y(t),t)=oo (or -oo),       |
    | 1**zoo       |         | but lim( x(t)**y(t), t) != 1.  See [3].       |
 
Only, in this case, we are able to give a more precise result: ``Limits(0, oo)``. (In fact, that could also be the value of ``1**oo``.)